development/containers/php/php-5.6.Dockerfile

```
@@ -0,0 +1,23 @@
+FROM php:5.6-cli-alpine AS php-base
+
+SHELL ["/bin/sh", "-euo", "pipefail", "-c"]
+
+RUN \
+   mv /usr/local/etc/php/php.ini-production /usr/local/etc/php/php.ini
+
+RUN \
```

Hadolint (2.15.1)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)